dask/dataframe/io/parquet.py

```diff
@@ -346,46 +538,33 @@ def to_parquet(df, path, compression=None, write_index=None, has_nulls=True,
     path : string
         Destination directory for data.  Prepend with protocol like ``s3://``
         or ``hdfs://`` for remote data.
+    engine : {'auto', 'fastparquet', 'pyarrow'}, default 'auto'
+        Parquet library to use. If only one library is installed, it will use
+        that one; if both, it will use 'fastparquet'
     compression : string or dict
```

Comment:
Not changed in this PR but out of curiosity: Should `compression=None` mean that the engine's default compression should be applied or no compression at all? (The default behaviour in the Parquet world is normally to have snappy enabled).

Reply:
I'd expect `None` to mean "no compression", as forwarding the compression overrides the default setting.

Perhaps we should change our default? If it's interpretted as "use the default setting for the backend", the default should maybe be `compression='default'` or something similar. I also wouldn't be opposed to changing it to snappy for fastparquet as well (thus matching pandas behavior), but @martindurant probably has more thoughts.

Reply:
I am totally fine, in theory, to have snappy be the default compression for fastparquet. The reason that it isn't is that snappy itself is only an optional dependency, one that might be tricky to build on some systems. I imagine that this location is inconvenient for checking of the importability of snappy.

Reply:
In dask.distributed we perform a little dance to determine default compression in a way that is robust to missing imports. https://github.com/dask/distributed/blob/master/distributed/protocol/compression.py

Reply:
Perhaps for now switch the dask kwarg default to `'default'`, which means the default compression for whatever backend is selected, and handle unification of this at a later point?